_posts/2024-05-14-switch-to-bam.md

```diff
@@ -0,0 +1,181 @@
+---
+layout: post
+title: What does it take to switch over to bam from fastq?
+tags:
+  - bam
+  - fastq 
+contributors:
+  - lskatz
+  - SeanSierra-Patev
+reviewers:
+  - pmenzel
+  - hexylena 
+---
+
+It has been over 14 years since the formalization of the fastq format ({% cite Cock2009 %}),
+which describes sequences and their quality scores.
+For paired end reads, sequences are encoded in separate files, usually called R1 and R2 [^1].
+Unfortunately despite the publication,
+fastq format is not entirely standardized!
+For example, it is possible to have a valid fastq format in either 4-line-per-entry format,
+or splitting sequences into multiple lines.
+Additionally, the defline itself is not entirely standardized and is basically free text.
+However, we have had a lot of innovations in sequence formats since then.
+One of those innovations is the BAM format which is the binary alignment/mapping format.
```

Comment:
```suggestion
One of those innovations is the [SAM/BAM format](https://samtools.github.io/hts-specs/), the (binary) alignment/mapping format. While this file format is typically used to store information about alignments of sequencing reads, it can also just stored the unaligned sequencing data.
```